src/interface/ContextInterface.tsx

```diff
@@ -61,6 +66,11 @@ export function retrieveInfoFromURLParameters(): boolean {
 
 export async function updateContexts(): Promise<boolean> {
   const strategy = await getSettings(AppSettings.contextEndpoint);
+  if (!AppSettings.applicationContext)
+    AppSettings.applicationContext = `${parsePrefix(
+      "a-popis-dat-pojem",
+      "aplikační-kontext"
+    )}/${uuidv4()}/ontographer`;
```

Comment:
I am not sure that we want to migrate the context links to "a-popis-dat" namespace.

Ideally we would use the "a-popis-dat" just to describe data and then perhaps use the existing "pracovní-prostor" to describe relations between contexts (those are not propagated to GitHub). But it's tricky nevertheless.

Reply:
Restored back to dsgov in the newest commit.